PeerioTechnologies/peerio-mobile
# feat: no keychain errors popups. retry instead [ch13622]

#### Relevant info and issue/PR links 

https://app.clubhouse.io/peerio/story/13622/ios-improved-handling-of-keychain-failure

#### Testing instructions  

Login successfully
Go to settings
Use either damage touch id or destroy touch id to corrupt the data there
Close app and open it again
You should be redirected to login welcome back screen
In case of destroy touch id you should have logs like this:

![image](https://user-images.githubusercontent.com/750359/45524249-79700b00-b79b-11e8-9fde-e4d699c2e4ae.png)


----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
